testing/mock/mock.go

```diff
@@ -23,6 +23,7 @@ import (
 const (
 	ModuleName = "mock"
 
+	PortID  = ModuleName
 	Version = "mock-version"
```

Comment:
Probably worth adding a line break here

```suggestion
	PortID  = ModuleName
	
	Version = "mock-version"
```